### PR TITLE
INTYGFV-14454: Add info text about chosen unit in care provider modal

### PR DESCRIPTION
**Pull request by HelenaEdblad**

### CODE REVIEW

File: packages/common/src/components/Table/ExpandableTableRow.tsx [hunk 1]
```diff
@@ -1,13 +1,19 @@
 import ArrowToggle from '@frontend/common/src/components/utils/ArrowToggle'
 import React, { Fragment, useState } from 'react'
+import styled from 'styled-components'
+
+const StyledButton = styled.button<{ disabled: boolean }>`
+  cursor: ${(props) => (props.disabled ? 'default' : 'pointer')};
```
Comment (mhornfeldt) [inline packages/common/src/components/Table/ExpandableTableRow.tsx:6]: Is this necessary when the button has the attribute disabled, shouldn't it come automatically?

**Review by mhornfeldt — CHANGES_REQUESTED** [review]

File: packages/common/src/components/Table/ExpandableTableRow.tsx (hunk 1 above)
Comment (HelenaEdblad) [reply packages/common/src/components/Table/ExpandableTableRow.tsx:6]: It didn't come automatically, that's why I added that css.

Comment (mhornfeldt) [reply packages/common/src/components/Table/ExpandableTableRow.tsx:6]: The default cursor for a disabled button is a pointer so I don't think you need to add logic for which cursor should be applied. If you have added som classnames which overrides the cursor: default styling then that should be the only thing you need to add.

Comment (HelenaEdblad) [reply packages/common/src/components/Table/ExpandableTableRow.tsx]: Ah, I removed the logic and added some classes instead now.

**Review by SamuelTilly — CHANGES_REQUESTED** [review]

File: packages/webcert/src/feature/careProvider/CareProviderModalContent.tsx [hunk 2]
```diff
@@ -63,27 +73,43 @@ export const CareProviderModalContent: React.FC = () => {
     }
   }
 
+  const getUnitName = (unit: Unit) => {
+    const chosenUnitInfoText = isLoggedInUnit(unit) && ' (vald enhet)'
+
+    if (isLoggedInUnit(unit)) {
+      return unit.unitName + chosenUnitInfoText
+    }
```
[the remark this answers is not in the document]
Comment (HelenaEdblad) [reply packages/webcert/src/feature/careProvider/CareProviderModalContent.tsx]: Oh, of course. Fixed!

**Review by SamuelTilly — APPROVED** [review]

**Pull request merged**